src/xdocs/property_types.xml

```diff
@@ -102,26 +102,33 @@
 
     <section name="regexp">
       <p>
-        This property represents a regular expression. The string
-        representation is parsed using <a
-        href="https://docs.oracle.com/javase/7/docs/api/index.html?java/util/regex/package-summary.html">java.util.regex
-        package</a>.
+        This type represents a regular expression. The string representation is parsed using
+        <a href="https://docs.oracle.com/javase/7/docs/api/index.html?java/util/regex/package-summary.html">
+        java.util.regex package</a>.
       </p>
     </section>
 
     <section name="uri">
       <p>
-        This property represents a URI. The string representation is parsed using a custom
+        This type represents a URI. The string representation is parsed using a custom
         routine to analyze what type of URI the string is.
         It can be a URL, regular file, or a resource path. It will try loading the path as
         a URL first, then as a file that must exist, and then finally as a resource on the
         classpath.
       </p>
     </section>
 
+    <section name="file">
```

Comment:
please move this update to separate commit that reference issue.
All other changes should be in separate commit, yuo can put "doc:" prefix for it.

Reply:
done